radicle-cli/src/git.rs

```diff
@@ -204,6 +248,15 @@ pub fn rad_remote(repo: &Repository) -> anyhow::Result<(git2::Remote, Id)> {
     }
 }
 
+pub fn rad_add_remote<'a>(
```

Comment:
I think this naming makes more sense because we're _adding_ the **rad** remote
```suggestion
pub fn add_rad_remote<'a>(
```

Reply:
I was thinking that there was a logic inside keep `rad_*` for git command for radicle?

I am fine if you want change this by I would like to keep the pattern `rad_*` because it is confusing `add_rad_remote` looks like we are adding our magic `rad` remote, but here we are adding any rad remote

Reply:
There's only one function that starts with `rad_` in that file, it's [`rad_remote`](https://github.com/radicle-dev/heartwood/blob/master/radicle-cli/src/git.rs#L197) because it finds the remote named `rad`.

So in fact, this function would be better named `add_remote`, since the remote could be any name :)